README.md

```diff
@@ -214,6 +214,8 @@ curl_easy_setopt(curl_handle, CURLOPT_CAINFO, "/etc/pki/tls/certs/ca-bundle.crt"
    - You can try hitting the non-TLS version of the endpoint if available. (Not Recommended).
 1. **No known conversion between `std::string` and `Aws::String`**
     - Either turn off custom memory management in the AWS C++ SDK or build it as a static library (`-DBUILD_SHARED_LIBS=OFF`)
+1. **I'm getting wierd linking errors like `/usr/bin/ld: /home/<...>bin/lib/libaws-lambda-runtime.a: error adding symbols: file format not recognized clang: error: linker command failed with exit code 1 (use -v to see invocation)`**
```

Comment:
@hbobenicio can you fix this type wierd -> weird.

Reply:
Woops :)
Nice catch!